<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>677</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>677</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>677</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>677</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HockeyScoreboardClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">:enabled
{
	background-color: rgb(46, 52, 54);
	selection-color: rgb(245, 121, 0);
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;Ubuntu Mono&quot;;
}
:disabled
{
	background-color: rgb(46, 52, 54);
	selection-color: rgb(245, 121, 0);
	color: rgb(136, 138, 133);
	font: 75 14pt &quot;Ubuntu Mono&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="firstTeamGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>661</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>661</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>251</width>
      <height>661</height>
     </size>
    </property>
    <property name="title">
     <string>Первая команда</string>
    </property>
    <widget class="QLineEdit" name="firstTeamNameLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{
	color: rgb(252, 175, 62);
	selection-background-color: rgb(252, 175, 62);
	selection-color: rgb(255, 255, 255);
}
:disabled
{
	color: rgb(136, 138, 133);
	selection-background-color: rgb(252, 175, 62);
	selection-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Team1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="firstTeamNameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Имя команды</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="firstTeamScoreTextLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Количество очков</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="firstTeamScoreValueLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>231</width>
       <height>400</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{
	font: 75 140pt &quot;Ubuntu Mono&quot;;
	color: rgb(252, 175, 62);
}
:disabled
{
	font: 75 140pt &quot;Ubuntu Mono&quot;;
	color: rgb(136, 138, 133);
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="firstTeamScoreAddPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="firstTeamScoreSubtractPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Отнять</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="secondTeamGroupBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>251</width>
      <height>661</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>661</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>251</width>
      <height>661</height>
     </size>
    </property>
    <property name="title">
     <string>Вторая команда</string>
    </property>
    <widget class="QLineEdit" name="secondTeamNameLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{
	color: rgb(252, 175, 62);
	selection-background-color: rgb(252, 175, 62);
	selection-color: rgb(255, 255, 255);
}
:disabled
{
	color: rgb(136, 138, 133);
	selection-background-color: rgb(252, 175, 62);
	selection-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Team2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="secondTeamNameLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Имя команды</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="secondTeamScoreTextLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Количество очков</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="secondTeamScoreValueLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>231</width>
       <height>400</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>400</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{
	font: 75 140pt &quot;Ubuntu Mono&quot;;
	color: rgb(252, 175, 62);
}
:disabled
{
	font: 75 140pt &quot;Ubuntu Mono&quot;;
	color: rgb(136, 138, 133);
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="secondTeamScoreAddPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="secondTeamScoreSubtractPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>231</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Отнять</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="genralGroupBox">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>10</y>
      <width>264</width>
      <height>661</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>264</width>
      <height>661</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>264</width>
      <height>661</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Общее</string>
    </property>
    <widget class="QLabel" name="generalCurrentTimeTextLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>245</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Текущее время</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="generalCurrentTimeValueLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>245</width>
       <height>101</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>101</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>101</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{	
	color: rgb(252, 175, 62);
	font: 75 56pt &quot;Ubuntu Mono&quot;;
}
:disabled
{
	font: 75 56pt &quot;Ubuntu Mono&quot;;
	color: rgb(136, 138, 133);
}</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="generalPeriodTimeTextLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>245</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Время периода</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="generalPeriodTimeValueLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>245</width>
       <height>101</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>101</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>101</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{	
	color: rgb(252, 175, 62);
	font: 75 56pt &quot;Ubuntu Mono&quot;;
}
:disabled
{
	font: 75 56pt &quot;Ubuntu Mono&quot;;
	color: rgb(136, 138, 133);
}</string>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="generalGameStartPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>245</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Начать игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generalGameStopPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>245</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Закончить игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generalGamePausePushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>245</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Приостановить игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generalGameContinuePushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>245</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Продолжить игру</string>
     </property>
    </widget>
    <widget class="QLabel" name="generalPeriodTextLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>245</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>245</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>245</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Номер периода</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="generalPeriodValueLabel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>550</y>
       <width>183</width>
       <height>101</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>183</width>
       <height>101</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>183</width>
       <height>101</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">:enabled
{
	font: 75 72pt &quot;Ubuntu Mono&quot;;
	color: rgb(252, 175, 62);
}
:disabled
{
	font: 75 72pt &quot;Ubuntu Mono&quot;;
	color: rgb(136, 138, 133);
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="generalPeriodPreviousPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>31</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generalPeriodNextPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>223</x>
       <y>550</y>
       <width>31</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
